tshirts/tshirts.cs

```diff
@@ -16,6 +16,7 @@ class Tshirt {
             Debug.Assert(Size(37) == "S");

```

Comment:
Since there are 3 ranges, atleast 2 boundary cases and 3 in range values can be tested. In addition if there is a limit for small and large then boundary cases increase and out of range values could also be tested.

Test cases can be enhanced a little more +1